lab-jessica/test/client-test.js

```diff
@@ -0,0 +1,28 @@
+const assert = require('assert');
+const Client = require('../lib/client.js');
+
+describe('Client Module', function() {
+  describe('#Client', function() {
+    it('creates new client', function() {
+      const clientExists = function() {
+        const client = new Client();
+        if(client) {
+          return true;
+        }
+      };
+      assert.strictEqual(true, clientExists());
```

Comment:
If there were no constructor for Client then an error would throw when you try to instantiate on line 8, so this whole test is a bit of a false positive. 

What properties of the client do your expect it to have? Can you test that those exist rather than just testing that the client object exists?